bundles/config/org.eclipse.smarthome.config.xml/src/main/java/org/eclipse/smarthome/config/xml/osgi/XmlDocumentBundleTracker.java

```diff
@@ -58,7 +58,7 @@
  */
 public class XmlDocumentBundleTracker<T> extends BundleTracker<Bundle> {
 
-    public static final String THREAD_POOL_NAME = "file-processing";
+    public static final String THREAD_POOL_NAME = "xml-processing";
```

Comment:
I'd assume that you didn't want to revert this change with https://github.com/eclipse/smarthome/pull/4851/commits/ecb254c22bf15b00613f8f6c138a47047b815f88.